src/kv/committable_tx.h

```diff
@@ -44,14 +46,27 @@ namespace kv
       }
 
       auto e = store->get_encryptor();
-      auto entry_type = claims_digest.empty() ? EntryType::WriteSet :
-                                                EntryType::WriteSetWithClaims;
+      auto commit_nonce = e->get_commit_nonce({commit_view, version});
+      commit_evidence = fmt::format(
+        "ce:{}.{}:{}", commit_view, version, ds::to_hex(commit_nonce));
```

Comment:
Since the receipt is very much JSON already, would it be nicer if the comment evidence was in the format `{"tx_id": "2.3", "commit_nonce": "<some hash>"}`?

Reply:
The receipt aren't JSON, we only have a JSON sample but some applications want to use binary formats. This format is I think a good half-way house between binary formats and JSON, but I'm open to suggestions that are also compatible with both.

Reply:
Good point, I'd missed that the formatting of the receipt was up to the application. I guess we could install a formatting lambda on the store to specify the serialisation format - but not for this PR.